src/xFactor/Judge.java

```diff
@@ -0,0 +1,95 @@
+package xFactor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Judge extends Person {
+	private int maxFavourites;
+	private static int countInstances;
+	private int vote;
+	private boolean isSpecial;
+	private ArrayList<Participant> favourites = new ArrayList<Participant>(maxFavourites);
+	private HashMap<Participant, Integer> votes = new HashMap<Participant, Integer>();
+	public static int numberOfJudges = 4;
+	
+	public int getVote() {
+		return vote;
+	}
+	
+	public static ArrayList<Judge> getJudges() {
+		return Competition.judges;
+	}
+	
+	public static int getJudgesSize() {
+		return Competition.judges.size();
+	}
+	
+	public int getMaxFavourites() {
+		return maxFavourites;
+	}
+
+	public void setMaxFavourites(int maxFavourites) {
+		this.maxFavourites = maxFavourites;
+	}
+	
+	public boolean isSpecial() {
+		return isSpecial;
+	}
+	
+	public void setSpecial(boolean isSpecial) {
+		this.isSpecial = isSpecial;
+	}
+	
+	public Judge(String fName, String lName, int age, String gender, String birthPlace) {
+		super(fName, lName, age, gender, birthPlace);
+		
+		if(countInstances == numberOfJudges) {
+			throw new IllegalArgumentException("Judges must be " + numberOfJudges);
+		}
+
+		Competition.judges.add(this);
+		countInstances++;
+	}
+	
+	public void addFavourite(Participant participant) {
+		if(favourites.size() < maxFavourites) {
+			for(Participant part : favourites) {
+				if(participant.equals(part)) {

```

Comment:
there are different collection types, which will prevent adding the same value multiple times - see Set